composer.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "pronovix/apigee_edge_ui",
```

Comment:
If we would like to publish this on Drupal.org is should be `drupal/`. I only saw one other non-PX module that was not `drupal/` something. I am not sure how Drupal Packagist would handle this, maybe it automatically replaces whatever is in `name` to ensure `composer require drupal/apigee_edge_ui` actually return this module.

Or we can just leave a note on this module's Drupal.org page how it can be installed from Github :) Which is quite simple and easy with Composer.

Reply:
Hmm. I'm not sure if this prefix states which packagist repository to use – but I'm almost sure that we don't have one internally.